roles/ec2-ami-base/files/tests/goss.yaml

```diff
@@ -0,0 +1,243 @@
+    groups:
+    - nogroup
+    home: /var/run/sshd
+    shell: /usr/sbin/nologin
+group:
+  docker:
+    exists: true
+    gid: 999
+process:
+  sshd:
+    running: true
+
+file:
+  /etc/ssh/sshd_config:
+    exists: true
+    contains:
+      - "UseDNS no"
+      - "PermitRootLogin no"
+      - "GSSAPIAuthentication no"
+  /etc/ntp.conf:
+    exists: true
+    contains:
+      - "tinker panic 0"
+  /etc/default/ntp:
+    exists: true
+    contains:
+      - 'NTPD_OPTS="-g -4"'
+  /etc/sysctl.conf:
+    exists: true
+    contains:
+      - 'vm.swappiness=10'
+      - 'vm.vfs_cache_pressure=50'
+  /etc/update-motd.d/10-help-text:
+    exists: false
+  /etc/update-motd.d/51-cloudguest:
+    exists: false
+  /etc/update-motd.d/90-updates-available:
+    exists: false
+  /etc/update-motd.d/91-release-upgrade:
+    exists: false
+  /etc/update-motd.d/20-hwe-eol:
+    exists: false
+  /etc/update-motd.d/98-fsck-at-reboot:
+    exists: false
+  /etc/update-motd.d/98-reboot-required:
+    exists: false
+  /etc/motd.tail:
+    exists: false
+  /etc/motd:
+    exists: false
+  /etc/modprobe.d/blacklist-ipv6.conf:
+    exists: true
+    contains:
+      - 'alias net-pf-10 off'
+      - 'alias ipv6 off'
+      - 'install ipv6 /bin/true'
+      - 'blacklist ipv6'
+  /etc/default/grub:
+    exists: true
+    contains:
+      - 'ipv6.disable=1'
+      - 'swapaccount=1'
+      - 'cgroup_enable=memory'
+  /root/.aws/config:
+    exists: true
+    owner: root
+    group: root
+  /home/ubuntu/.aws/config:
+    exists: true
+    owner: ubuntu
+    group: ubuntu
+  /etc/salt/minion_id:
+    exists: true
+    owner: root
+    group: root
+  /etc/salt/pki/minion/:
+    exists: true
+    owner: root
+    group: root
+  /etc/salt/minion:
+    exists: true
+    contains:
+      - "tcp_keepalive: True"
+      - "tcp_keepalive_idle: 300"
+  /usr/local/bin/docker-compose:
+    exists: true
+    owner: root
+    group: root
+  /etc/fstab:
+    exists: true
+    owner: root
+    group: root
+    contains:
+      - /srv/docker /var/lib/docker none bind 0 0
+  /srv/docker:
+    exists: true
+    owner: root
+    group: root
+  /var/lib/docker:
+    exists: true
+    owner: root
+    group: root
+  /etc/default/ufw:
+    exists: true
+    owner: root
+    group: root
+    contains:
+      - DEFAULT_INPUT_POLICY="ACCEPT"
+      - DEFAULT_OUTPUT_POLICY="ACCEPT"
+      - DEFAULT_FORWARD_POLICY="ACCEPT"
+  /etc/hosts:
+    exists: true
+    owner: root
+    group: root
+    contains:
+      - 127.0.0.1 localhost.localdomain localhost loopback
+  /etc/skel/.bashrc_opg:
+    exists: true
+    owner: root
+    group: root
+  /etc/skel/.bashrc:
+    exists: true
+    owner: root
+    group: root
+    contains:
+      - source /etc/skel/.bashrc_opg
+  /etc/os-release-ec2:
+    exists: true
+    owner: root
+    group: root
+    contains:
+      - BUILD_NUMBER=
+      - AMI_SERIAL_NUMBER=
+  /etc/cron.daily/apt:
+    exists: false
+  /etc/cron.daily/apt-compat:
+    exists: false
```

Comment:
:+1:

In fact ![image](https://media.giphy.com/media/E0cyxhawhe9dm/giphy.gif) because you earned it